uisdk/build.gradle.kts

```diff
@@ -1,4 +1,4 @@
-version = "1.5.1"
+version = "1.6.0"
```

Comment:
can we do that only when we start the release as this will update the read me and we didn't yet released